kubera/authentication/authentication.go

```diff
@@ -0,0 +1,111 @@
+package authentication
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/go-resty/resty/v2"
+	conn "github.com/mayadata-io/kubera-api-testing/kubera/connection"
+)
+
+type Token struct {
+	JWT       string `json:"jwt"`
+	AccountID string `json:"accountId"`
+}
+
+func (t Token) String() string {
+	raw, err := json.MarshalIndent(
+		t,
+		" ",
+		".",
+	)
+	if err != nil {
+		panic(err)
+	}
+	return string(raw)
+}
+
+type APIError struct {
```

Comment:
can we have a comment that what the below structure will be doing?

Reply:
will make these changes in next PR  🗒️